server/core/region_tree.go

```diff
@@ -26,12 +26,151 @@ import (
 	"go.uber.org/zap"
 )
 
+// RegionItem is region tree item.
+type RegionItem interface {
```

Comment:
RegionItem -> RangeItem
RegionTree -> RTree or `RangeTree`  or else?

better move this part to `pkg`?

Reply:
fix